diff --git "a/SourceFiles\\resource.h" "b/Win32Explorer_26.0.3.0\\App_Source\\EliteTaskbar\\resource.h"
index fdf427d..e2c9df6 100644
--- "a/SourceFiles\\resource.h"
+++ "b/Win32Explorer_26.0.3.0\\App_Source\\EliteTaskbar\\resource.h"
@@ -1,4 +1,5 @@
 #pragma once
+#pragma warning(disable: 4715)
 
 #define IDI_MAIN_PROGRAM 101
 #define IDI_PREFERENCES  102
@@ -77,3 +78,4 @@
 #define IDC_ABOUT_MOREINFO           283
 #define IDC_HELP_TEXT                284
 #define IDC_BANNER                   285
+// Modified for Milestone 2 settings updates
